pwndbg/commands/heap.py

```diff
@@ -704,10 +717,33 @@ def vis_heap_chunks(addr=None, count=None, naive=None):
 
     cursor = cursor_backup
 
+    has_huge_chunk = False
+    # round up to align with 4*ptr_size and get half
+    half_max_size = ((max_visualize_chunk_size - 1) // (ptr_size << 2) + 1) * (ptr_size << 2) // 2
+
     for c, stop in enumerate(chunk_delims):
         color_func = color_funcs[c % len(color_funcs)]
 
+        if stop - cursor > 0x10000:
```

Comment:
I'm not too familiar with this code, but what is this `0x10000` constant for exactly?

Reply:
`stop - cursor` is the current chunk size.